Conceptpower+Spring/src/main/webapp/WEB-INF/views/auth/concepts/MergeConcepts.jsp

```diff
@@ -0,0 +1,92 @@
+<%@ taglib uri="http://www.springframework.org/tags" prefix="spring"%>
+<%@ taglib prefix="sec"
+    uri="http://www.springframework.org/security/tags"%>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core"%>
+<%@ taglib prefix="form" uri="http://www.springframework.org/tags/form"%>
+<head>
+<title>Merge Concept</title>
+<form:form action="${pageContext.servletContext.contextPath}/auth/concepts/merge" method='post' modelAttribute="conceptsMergeBean">
+
+    <h1>Merge concepts</h1>
+
+    <table>
+        <tr>
+            <td class="col-sm-1"> Concept Id </td>
+        </tr>
+        <c:forEach var="conceptId" items="${conceptsMergeBean.conceptIds}">
```

Comment:
Isn't there an input tag radiobuttons (that prints all of them)? Also, separating the radio button and its label in two different tds seems (without having tested it yet) like it might look a little funny on big screens (tds being pulled apart).